### PR TITLE
Update DB Entities and add seed data

### PR DESCRIPTION
**Pull request by onb6**
This PR has a bunch of random changes that I made as I was creating seed data for the database: 
- Added control/treatment enum to youth entity
- Added uuid to assignment entity
- Changed "completed" boolean on assignment to be Status enum
- Created seed class to insert data into all entities (except question and option because we are using defaults for now)
- Created seed script to run the seeds in the right order in order to allow for foreign key relationships
- Updated readme with seed scripts

### CODE REVIEW

File: migrations/1640012056080-add-reviewer-uuid.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class addReviewerUuid1640012056080 implements MigrationInterface {
```
Comment (rymaju) [inline migrations/1640012056080-add-reviewer-uuid.ts:3]: Were these migration files not added before? :yikes:

File: package.json [hunk 2]
```diff
@@ -31,7 +31,8 @@
     "seed:config": "ts-node ./node_modules/typeorm-seeding/dist/cli.js config",
     "seed:run": "ts-node ./node_modules/typeorm-seeding/dist/cli.js seed",
     "schema:drop": "ts-node ./node_modules/typeorm/cli.js schema:drop",
-    "schema:sync": "ts-node ./node_modules/typeorm/cli.js schema:sync"
+    "schema:sync": "ts-node ./node_modules/typeorm/cli.js schema:sync",
+    "seed": "./seedDatabase.sh"
```
Comment (rymaju) [inline package.json:35]: This might not work on windows 🤔 I'll try it locally

File: src/assignment/types/assignment.entity.ts [hunk 3]
```diff
@@ -28,9 +31,11 @@ export class Assignment {
   youth: Youth;
 
   @Column({
-    default: false,
+    type: 'enum',
+    enum: AssignmentStatus,
```
Comment (rymaju) [inline src/assignment/types/assignment.entity.ts:35]: enums 😍

File: src/seeds/seed-reviewers.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+export default class CreateReviewers implements Seeder {
+  public async run(factory: Factory, connection: Connection): Promise<any> {
+    await connection
+      .createQueryBuilder()
+      .insert()
+      .into(Reviewer)
+      .values([
+        {
+          email: 'c4cneu.jpal+ben.lerner@gmail.com',
+          firstName: 'Ben',
```
Comment (rymaju) [inline src/seeds/seed-reviewers.ts:14]: I question putting real people's names in but both of these people have hurt me in some significant way so I dont mind

**Review by rymaju — APPROVED** [review]
Looks great! I need to test it locally but this will make backend work a lot easier!

One big nitpick ⛏️ is that I wish that the example data being inserted into the DB was extracted into some examples files. That way we can easily reuse this data for E2E tests.

File: migrations/1640012056080-add-reviewer-uuid.ts (hunk 1 above)
Comment (onb6) [reply migrations/1640012056080-add-reviewer-uuid.ts:3]: There was no reviewer uuid before this, I just added it!

File: package.json (hunk 2 above)
Comment (rymaju) [reply package.json:35]: image
tragic

File: migrations/1640012056080-add-reviewer-uuid.ts (hunk 1 above)
Comment (onb6) [reply migrations/1640012056080-add-reviewer-uuid.ts:3]: oh wait. yes there was

Comment (onb6) [reply migrations/1640012056080-add-reviewer-uuid.ts:3]: thats a big :yikes:

File: README.md [hunk 5]
```diff
@@ -46,4 +50,8 @@ Example usage: `npm run typeorm migration:generate -- -n "createdUserProfiles"`
 
 ### `npm run schema:drop`
 
-Drops your database. Do not do this unless you really need to!
+Drops your database. Do not do this unless you really need to! Does not reinitialize the database with a schema. If you want to just reset things use `npm run reset-db` or `npm run schema:drop && npm run migration`.
+
+### `npm run reset-db`
```
Comment (onb6) [inline README.md:55]: this is lovely :,)

File: package.json [hunk 6]
```diff
@@ -31,7 +31,9 @@
     "seed:config": "ts-node ./node_modules/typeorm-seeding/dist/cli.js config",
     "seed:run": "ts-node ./node_modules/typeorm-seeding/dist/cli.js seed",
     "schema:drop": "ts-node ./node_modules/typeorm/cli.js schema:drop",
-    "schema:sync": "ts-node ./node_modules/typeorm/cli.js schema:sync"
+    "schema:sync": "ts-node ./node_modules/typeorm/cli.js schema:sync",
+    "seed": "npm run seed:run -- -s \"CreateUsers\" && npm run seed:run -- -s \"CreateReviewers\" && npm run seed:run -- -s \"CreateYouth\" && npm run seed:run -- -s \"CreateSurveyTemplates\" && npm run seed:run -- -s \"CreateSurveys\" && npm run seed:run -- -s \"CreateAssignments\"",
```
Comment (onb6) [inline package.json:35]: i hate that this is the only way, but if it works it works

Comment (rymaju) [reply package.json:35]: 286 columns of awful 😞

**Comment by onb6** [comment]
@rymaju I made example data files for reviewer, youth, and users, but I am not sure how to handle it for the others because they have foreign keys. They wouldn't have access to the connection and im not sure how else we can reference the other tables. I think we should just leave with examples only for those for now

**Comment by rymaju** [comment]
> @rymaju I made example data files for reviewer, youth, and users, but I am not sure how to handle it for the others because they have foreign keys. They wouldn't have access to the connection and im not sure how else we can reference the other tables. I think we should just leave with examples only for those for now

Excellent point that I completely forgot about. You're right, lets not for now

**Review by michelle-vel — APPROVED** [review]
:))

**Pull request merged**